.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow deploys the master brance to the dev site
+
+name: dev-deployment
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to deploy'
+        required: false
+        default: 'master'
+  # Triggers the workflow on push events but only for the master branch
+  push:
+    branches: [ master ]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: Configure gcloud
+        run: gcloud config set compute/zone us-central1-c
+      - name: Run deploy script
+        run: |
+          gcloud compute ssh openreview@instance-matching-server --command '/bin/matcher_build_dev.sh -b ${{ github.event.inputs.branch }}' --quiet
```

Comment:
what does this variable mean? 
```
${{ github.event.inputs.branch }}
```

Reply:
If we want to deploy a particular branch on dev, we can pass the branch name.

Reply:
Intereting. Can you show me an example of how this would be done?

Reply:
If you go to the Actions tab and check the dev-deployment workflow, there should be an option of run-workflow there.

Reply:
Ah, I understand now. That's a nice feature, I'm assuming that the default value is master and that if it's changed it will only be for that particular run of the workflow. Subsequent executions of the workflow will default back to master branch, correct?

Reply:
Yes, that's correct.

Reply:
Awesome